zulipterminal/ui_tools/messages.py

```diff
@@ -227,9 +227,9 @@ def top_search_bar(self) -> Any:
                     [(bar_color, self.stream_name)],
                 )
         elif len(curr_narrow) == 1 and len(curr_narrow[0][1].split(",")) > 1:
-            text_to_fill = "Group private conversation"
+            text_to_fill = "Group direct conversation"
         else:
-            text_to_fill = "Private conversation"
+            text_to_fill = "Direct conversation"
```

Comment:
This phrasing doesn't end up as good as before with a substitution - a 'direct conversation'?

Perhaps for now have `private`->`direct message` in these two cases.

The underlying problem is that we don't move the current message recipient text into this area and hide the current message recipients, when we're "fully narrowed", ie. we're just showing one conversation and not "interleaved" messages with headers for different topics, direct-messages, etc.

Reply:
Yeah, `direct message conversation` sounds better than `direct conversation`. I'll make the change.

>The underlying problem is that we don't move the current message recipient text into this area and hide the current message recipients, when we're "fully narrowed", ie. we're just showing one conversation and not "interleaved" messages with headers for different topics, direct-messages, etc.

Comparing it to how zulip web handles it, the web version replaces the search header text with the recipient name, while keeping the `You and Some One` header. Maybe we can change it to be the same in ZT?

Reply:
Yeah, there have definitely been conversations about this in the past. I'm fairly sure we don't have an open PR for this, but there may be an issue, and if you can dig out an older topic about this, this may help with understanding if there was a conclusion reached.